CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+- Added: Generate meta description and bundle it to collection
```

Comment:
Can you add a pointer for people that will upgrade? Like adding:

```
See ``boilerplate/web_modules/layouts/Page/index.js`` for an example.
```

Where do you think we should add doc for this?